channel/funder_test.go

```diff
@@ -94,10 +94,10 @@ func newFunderSetup(rng *rand.Rand) (
 		accs[i] = accounts.Account{Address: ethwallet.AsEthAddr(wallettest.NewRandomAddress(rng))}
 	}
 	// Use an ETH depositor with random addresses at index 0.
-	depositors[0] = ethchannel.NewETHDepositor()
+	depositors[0] = ethchannel.NewETHDepositor(50000)
 	// Use an ERC20 depositor with random addresses at index 1.
 	token := wallettest.NewRandomAddress(rng)
-	depositors[1] = ethchannel.NewERC20Depositor(ethwallet.AsEthAddr(token))
+	depositors[1] = ethchannel.NewERC20Depositor(ethwallet.AsEthAddr(token), 100000)
```

Comment:
This gaslimit value is already defined above as txGasLimit. You can use that value.
In fact, both depositors, the NewETHDepositor and the NewERC20Depositor could use constants defined above: Instead of txGasLimit, you can set txERC20GasLimit and txETHGasLimit for both cases, with 100000 and 50000, respectively.